test/integration/util/util.go

```diff
@@ -63,22 +63,7 @@ func StartScheduler(clientSet clientset.Interface, enableEquivalenceCache bool)
 	evtWatch := evtBroadcaster.StartRecordingToSink(&clientv1core.EventSinkImpl{
 		Interface: clientv1core.New(clientSet.CoreV1().RESTClient()).Events("")})
 
-	schedulerConfigurator := factory.NewConfigFactory(
```

Comment:
passing the informerFactory in makes it too easy to save a handle to the informerFactory and try to set up shared informers after config setup time. enumerating the informers required here is preferred.

Reply:
Good catch, will do it, thanks for pointing this out!

Reply:
Fixed. While I refactored this into a new function. This should not be a problem now.